.gitignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
+*.sw[op]
+*.pyc
+__pycache__
+.cache
+.tox
+*.json
+*.asc
+log.txt
+run.sh
```

Comment:
We're going to move the build.sh and [uncommitted] run.sh files into the new and improved README.md file. 🎉